.evg.yml

```diff
@@ -339,10 +439,11 @@ buildvariants:
     realm_server_url: "http://localhost:9090"
     yarn_url: "https://s3.amazonaws.com/stitch-artifacts/yarn/latest.tar.gz"
     node_version: 12.16.2
-    version: 2.3.0
+    cli_version: 2.3.0
```

Comment:
what was this renamed for?

Reply:
"grep-ability" ... basically this bit:

```cmd
cat .evg.yml | grep cli_version: | awk '{ print $2 }' | tail -n 1
```

the `tail -n 1` worked for now in grabbing the `version: ` field I was looking for, but this felt a bit more future-proof